packages/block-library/src/quote/index.js

```diff
@@ -100,18 +100,39 @@ export const settings = {
 			},
 			{
 				type: 'raw',
-				isMatch: ( node ) => (
-					node.nodeName === 'BLOCKQUOTE' &&
+				isMatch: ( node ) => {
+					return node.nodeName === 'BLOCKQUOTE' &&
 					// The quote block can only handle multiline paragraph
-					// content.
-					Array.from( node.childNodes ).every( ( child ) => child.nodeName === 'P' )
-				),
+					// content with an optional cite child.
+					Array.from( node.childNodes ).every(
```

Comment:
How about naming this IIFE after what it does and write this code like:

```
return node.nodeName === 'BLOCKQUOTE' &&
    Array.from( node.childNodes ).every( isParagraphOrSingleCite() );
```

I find this code more readable.